packages/office-addin-project/README.md

```diff
@@ -0,0 +1,28 @@
+# Office-Addin-Manifest
+
+This package provides the ability to do project wide commands for Office Add-ins, such as conversion.
+
+For more information, see the [documentation](
+https://docs.microsoft.com/en-us/office/dev/add-ins/develop/add-in-manifests).
+
+## Command-Line Interface
+* [convert](#info)
+
+#
+
+### convert
+
+Converts the Office Addin Code.
```

Comment:
Convert it t6o what?

Reply:
Not sure what to put here yet. I will just mention from XML to json manifest for now.
We will probably just publish this package when we will convert more stuff.